plugins/autopeering/autopeering.go

```diff
@@ -136,25 +132,21 @@ func parseEntryNodes() (result []*peer.Peer, err error) {
 		if len(parts) != 2 {
 			return nil, fmt.Errorf("%w: entry node parts must be 2, is %d", ErrParsingEntryNode, len(parts))
 		}
-		pubKey, err := base64.StdEncoding.DecodeString(parts[0])
-		if err != nil {
-			return nil, fmt.Errorf("%w: can't decode public key: %s", ErrParsingEntryNode, err)
-		}
 
-		entryAddr, err := iputils.ParseOriginAddress(parts[1])
+		pubKey, err := base64.StdEncoding.DecodeString(parts[0])
 		if err != nil {
-			return nil, fmt.Errorf("%w: invalid entry node address %s", err, parts[1])
+			return nil, fmt.Errorf("%w: invalid public key: %s", ErrParsingEntryNode, err)
 		}
 
-		ipAddresses, err := iputils.GetIPAddressesFromHost(entryAddr.Addr)
+		addr, err := net.ResolveUDPAddr("udp", parts[1])
```

Comment:
This is handled a bit differently as before. Before iputils was used to get the IP address while no the `net` is used directly. Should be more or less equivalent, the only potential difference I can see is that in this version, no longer considers `config.CfgNetPreferIPv6` when the host can be resolved to ipv4 and ipv6. But I am not sure this actually makes a difference